harness/determined/common/experimental/experiment.py

```diff
@@ -274,6 +280,25 @@ def unarchive(self) -> None:
         bindings.post_UnarchiveExperiment(self._session, id=self._id)
         self.archived = False
 
+    def move_to_project(self, workspace_name: str, project_name: str) -> None:
+        """Move an experiment to a different project.
```

Comment:
"""Transfer the experiment to another project.

Reply:
"move" is consistent with both the CLI's `det experiment move` command and the web UI. All other things being equal, it's nice to minimize changes between the two. Where does this recommendation for "transfer" instead of "move" come from?

Reply:
let's keep "move" since it is consistent with the CLI

(i was thinking of "moving" as motion and "transferring" as moving from one place to another)